gym_game/envs/connectFour.py

```diff
@@ -2,13 +2,19 @@
 import numpy as np 
 
 class connect_four():
-    def __init__(self, game_size: int):
+    def __init__(self):
         # The attributes to display the game
-        self.size = game_size
+        self.size = 1000
```

Comment:
What is the reasoning behind this change? Do you even use this attribute for anything?

Reply:
Use it for the size of the window, when we want to display the game being played.

Changed it so it is not a parameter you can give to the game, but rather something we decide inside the game object